build-system/tasks/e2e/build-e2e.js

```diff
@@ -0,0 +1,126 @@
+const {compileCss} = require('../css');
+const {
+  buildBinaries,
+  buildExtensions,
+  buildNpmCss,
+} = require('../extension-helpers');
+const {mkdirSync} = require('../helpers');
+const fastGlob = require('fast-glob');
+const {endBuildStep} = require('../helpers');
+const path = require('path');
+const fs = require('fs-extra');
+
+/**
+ * Build the Bento end to end page
+ *
+ * @param {!Object} options
+ * @return {!Promise<void>}
+ */
+async function buildBentoE2E(options) {
```

Comment:
Not too familiar with how bento is currently built, but is there existing code that can be reused here?

Reply:
Like this? https://github.com/ampproject/amphtml/blob/ca6d5d66f38d74e905f14b075a7dd70a09344f20/build-system/tasks/dist.js#L138

Reply:
+1

Reply:
piling on: ideally theres only a single `buildBentoComponent` function that builds both the react/preact versions

Reply:
This one's context is a big different. I don't want the e2e fixtures to build everytime we want to build the extensions. And I'm not a big fan of extending the `buildExtentions` and `buildBentoComponents` to also be aware of the building the e2e fixtures. This would break single responsibility

Reply:
re: different contexts - ideally the components that are built for e2e tests are as close as possible to what is released. If we have a separate build function, there's the possibility of diverging from prod, which could lead to problems down the road.

re: not building every time - we can conditionally build by checking `argv.nobuild` just as we do for the runtime. What do you think?

Reply:
It doesn't run every time, only when we run the e2e tests. I don't think I want the e2e fixtures to be built on every dist or build command when they're not needed.

Reply:
Let's discuss the `nobuild` option here - https://github.com/ampproject/amphtml/pull/37626#discussion_r821030159

My initial comment still stands though. Can we avoid having a separate build function?